### ISSUE TITLE
Docker files?

### ISSUE THREAD

**Issue by janvidar** [jensl/critic#119]
Hi,

You should really look into creating docker files so it is easy to install this thing in a reproducible manner without the need to manually configure all the different dependencies as a user.

Any plans for that?

**Comment by jensl** [comment]
Not really. Many other things to do, and I don't know Docker, so it's not something I can without effort.

Needless to say, I would consider it very awesome if someone else got the "project" started for me.

**Comment by janvidar** [comment]
Ah, OK.
I just found this though: https://hub.docker.com/r/tjarosik/critic/

**Comment by Zitrax** [comment]
Based loosely on the link above I have a Dockerfile that in addition builds and adds extensionsupport. It's not very much tested but it builds and launches. It's just in a gist currently: https://gist.github.com/Zitrax/0373405ba0b7a01d75afd18da223fdd7
